ec-service/src/main/java/org/dcsa/endorsementchain/unofficial/mappers/TransactionMapper.java

```diff
@@ -0,0 +1,14 @@
+package org.dcsa.endorsementchain.unofficial.mappers;
+
+import org.dcsa.endorsementchain.persistence.entity.Transaction;
+import org.dcsa.endorsementchain.transferobjects.EndorsementChainTransaction;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper(componentModel = "spring")
+public interface TransactionMapper {
+
+  @Mapping(expression = "java(System.currentTimeMillis())", target = "timestamp")
+  @Mapping(source = "platformName", target = "platformHost")
+  Transaction EndorsementChainTransactionToTransaction(EndorsementChainTransaction endorsementChainTransaction, String platformName);
```

Comment:
`endorsementChainTransactionToTransaction` to follow java naming conventions?

Reply:
fixed